packages/fuels-test-helpers/src/node.rs

```diff
@@ -281,10 +281,11 @@ pub async fn new_fuel_node(
 pub async fn server_health_check(client: &FuelClient) {
     let mut attempts = 5;
     let mut healthy = client.health().await.unwrap_or(false);
+    let between_attempts = Duration::from_millis(300);
```

Comment:
Any reason we're increasing this to 300?

Also, unrelated, but: we should pull this from an env var.

Reply:
Previous retry period was too rapid. It seems the node takes longer to become healthy. 300 is just empirically determined to produce repeatable ci runs.

I'll create a task to make it configurable tomorrow.